projects/ion/src/lib/dropdown/dropdown.component.spec.ts

```diff
@@ -497,3 +497,55 @@ describe('IonDropdownComponent / Required', () => {
     expect(document.getElementById('option-0')).toHaveClass('dropdown-item');
   });
 });
+
+describe('IonDropdownComponent / No data', () => {
+  const defaultNoData = {
+    ...defaultDropdown,
+    options: [],
+    selected: {
+      emit: selectEvent,
+    } as SafeAny,
+  };
+
+  it('Should render NoData when there are no options.', async () => {
+    await sut(defaultNoData);
+    const element = screen.getByTestId('ion-no-data');
+    expect(element).toBeInTheDocument();
+    expect(screen.getByText('Não há dados')).toBeInTheDocument();
+    expect(
+      document.getElementById('ion-icon-exclamation-rounded')
+    ).toBeInTheDocument();
+  });
+
+  it('Should not render NoData when there are options.', async () => {
```

Comment:
I believe this test is not necessary, as there are already tests that validate this

Reply:
ok, I'm going to delete the it that is repeating the validation of the default parameters

Reply:
I did the test to validate if noData was rendered with the default parameters because the storybook wasn't showing, I had to manually pass the noData configuration in the story.